docs/markdown/Builtin-options.md

```diff
@@ -25,21 +25,31 @@ Installation options are all relative to the prefix, except:
-| ------         | ------------- | -----------
-| prefix         | see below     | Installation prefix
-| libdir         | see below     | Library directory
-| libexecdir     | libexec       | Library executable directory
-| bindir         | bin           | Executable directory
-| sbindir        | sbin          | System executable directory
-| includedir     | include       | Header file directory
-| datadir        | share         | Data file directory
-| mandir         | share/man     | Manual page directory
-| infodir        | share/info    | Info page directory
-| localedir      | share/locale  | Locale data directory
-| sysconfdir     | etc           | Sysconf data directory
-| localstatedir  | var           | Localstate data directory
-| sharedstatedir | com           | Architecture-independent data directory
+| Option                               | Default value | Description |
+| ------                               | ------------- | ----------- |
+| prefix                               | see below     | Installation prefix |
+| libdir                               | see below     | Library directory |
+| libexecdir                           | libexec       | Library executable directory |
+| bindir                               | bin           | Executable directory |
+| sbindir                              | sbin          | System executable directory |
+| includedir                           | include       | Header file directory |
+| datadir                              | share         | Data file directory |
+| mandir                               | share/man     | Manual page directory |
+| infodir                              | share/info    | Info page directory |
+| localedir                            | share/locale  | Locale data directory |
+| sysconfdir                           | etc           | Sysconf data directory |
+| localstatedir                        | var           | Localstate data directory |
+| sharedstatedir                       | com           | Architecture-independent data directory |
+| werror                               | false         | Treat warnings as erros |
+| warnlevel {1, 2, 3}                  | 1             | Set the warning level. From 1 = lowest to 3 = highest |
+| layout {mirror,flat}                 | mirror        | Build directory layout. |
+| default-library {shared, static, both} | shared       | Default library type. |
+| backend {ninja, vs,<br>vs2010, vs2015, vs2017, xcode} |               | Backend to use (default: ninja). |
+| stdsplit                             |               | Split stdout and stderr in test logs. |
+| errorlogs                            |               | Whether to print the logs from failing tests. |
+| cross-file CROSS_FILE                |               | File describing cross compilation environment. |
+| wrap-mode {WrapMode.default,<br>WrapMode.nofallback, WrapMode.nodownload,<br>WrapMode.forcefallback} | | Special wrap mode to use |
```

Comment:
This should be `{default, nofallback, nodownload, forcefallback}`.

Also the table does not need to end with `|` afaik.

Reply:
The table must end with |

Reply:
Okay :)